### ISSUE TITLE
[HOLD for payment 2023-02-28] [$1000]  'You can also use the + button below to send or request money!' message for group chat

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#14764]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app in https://staging.new.expensify.com
2. Open any group (or create a group chat by clicking the '+' > New Group > add members)
3. Scroll to top of the chat
4. notice "You can also use the + button below to send or request money!" verbiage 
5. Click the '+' in the chat and notice there is not Send or Request money option

## Expected Result:
'You can also use the + button below to send or request money!' should not be displayed in group chat because there's no method to request or send money in the group

![image](https://user-images.githubusercontent.com/51066321/216520597-68c8ccb8-0f2b-4114-8db2-e06bd88f13c2.png)
![image](https://user-images.githubusercontent.com/51066321/216520632-62116782-d982-4582-8d88-6761411842b4.png)

## Actual Result:
'You can also use the + button below to send or request money!' is displayed in group chat

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** v1.2.64-2
**Reproducible in staging?:** y
**Reproducible in production?:** new UI
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/216407592-7f69ac3d-851a-4456-b898-75f0263d2ece.mp4

<img width="960" alt="1" src="https://user-images.githubusercontent.com/43996225/216407666-19986b49-b009-4d70-8173-056f6b556718.png">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1675331757347979

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01858fd9dc0302d46c</li>
        <li>Upwork Job ID: 1621383713253961728</li>
        <li>Last Price Increase: 2023-02-03</li>
    </ul>
</details>

**Comment by Christinadobrzyn** [comment]
- I can reproduce this on MacOS / Chrome / Safari v1.2.64-4
- This doesn't seem to affect iOS / Safari / MacOS / Desktop / iOS / native
- Update the OP with additional information & screenshots
- I can't find another issue related to this
- This can be an external issue

**Comment by Tushu17** [comment]
We can use Operator to render this Text component when `participants.length < 2` or `participants.length === 1` :

https://github.com/Expensify/App/blob/117e03085ad3944ff4ebe491bc05ca8183093227/src/components/ReportWelcomeText.js#L126-L129

``` diff
                       {participants.length < 2 && (
                            <Text>
                                {props.translate('reportActionsView.usePlusButton')}
                            </Text>
                        ) }
```
### 2nd Option
We have Split Bill option in Group chat. We can♡ also use ternary operator where we can show different Text for group chat.

``` diff

                            <Text>
                                {participants.length < 2 ? props.translate('reportActionsView.usePlusButton') : 'Some other text'}
                            </Text>

``

**Comment by getusha** [comment]
### Proposal

```diff --git a/src/components/ReportWelcomeText.js b/src/components/ReportWelcomeText.js
index c732cc589d..5547c059e8 100644
--- a/src/components/ReportWelcomeText.js
+++ b/src/components/ReportWelcomeText.js
@@ -123,10 +123,12 @@ const ReportWelcomeText = (props) => {
                                 {(index < displayNamesWithTooltips.length - 2) && <Text>, </Text>}
                             </Text>
                         ))}
-                        <Text>
-                            {/* Need to confirm copy for the below with marketing, and then add to translations. */}
-                            {props.translate('reportActionsView.usePlusButton')}
-                        </Text>
+                        {!isMultipleParticipant &&
+                            <Text>
+                                {/* Need to confirm copy for the below with marketing, and then add to translations. */}
+                                {props.translate('reportActionsView.usePlusButton')}
+                            </Text>
+                        }
                     </Text>
                 )}
             </Text>
```

**Comment by bernhardoj** [comment]
### **Proposal**

I think we need to check if the participants equal to 2 and not a chat room.

```diff
diff --git a/src/components/ReportWelcomeText.js b/src/components/ReportWelcomeText.js
index c732cc589..9f87b604b 100644
--- a/src/components/ReportWelcomeText.js
+++ b/src/components/ReportWelcomeText.js
@@ -123,10 +123,10 @@ const ReportWelcomeText = (props) => {
                                 {(index < displayNamesWithTooltips.length - 2) && <Text>, </Text>}
                             </Text>
                         ))}
-                        <Text>
+                        {!isChatRoom && participants.length === 2 && <Text>
                             {/* Need to confirm copy for the below with marketing, and then add to translations. */}
                             {props.translate('reportActionsView.usePlusButton')}
-                        </Text>
+                        </Text>}
                     </Text>
                 )}
             </Text>
```

based on the condition here 
https://github.com/Expensify/App/blob/8f754df0ddea4249eb5d1f94ea6642a8a797d08c/src/pages/home/report/ReportActionCompose.js#L275-L295

**Comment by PankajAS** [comment]
I think we need to change text here because if we add send or request money option here then request/send money will also split in group members.

**Comment by tienifr** [comment]
# Proposal

## Problem

In https://github.com/Expensify/App/blob/8f754df0ddea4249eb5d1f94ea6642a8a797d08c/src/pages/home/report/ReportActionCompose.js#L287-L296
we're checking `ReportUtils.isChatRoom(this.props.report) || hasMultipleParticipants` to show split bill
but in `ReportWelcomeText` we lack of checking `isMultipleParticipant` to show usePlusButton

## Solution

In `ReportWelcomeText`

```diff

-                        <Text>
+                        {!isMultipleParticipant && <Text>
                            {/* Need to confirm copy for the below with marketing, and then add to translations. */}
                            {props.translate('reportActionsView.usePlusButton')}
+                        </Text>}
-                        </Text>

```

I also find one thing redundant and need to improve 

In https://github.com/Expensify/App/blob/8f754df0ddea4249eb5d1f94ea6642a8a797d08c/src/pages/home/report/ReportActionCompose.js#L526 
we're removing `this.props.currentUserPersonalDetails.login` from participants
but in https://github.com/Expensify/App/blob/8f754df0ddea4249eb5d1f94ea6642a8a797d08c/src/pages/home/report/ReportActionCompose.js#L276
we're still having the logic to remove `this.props.currentUserPersonalDetails.login`

Here's my suggestion

```diff

    /**
     * Returns the list of IOU Options
     *
-     * @param {Array} reportParticipants
+     * @param {Array} participants
     * @returns {Array<object>}
     */
-    getIOUOptions(reportParticipants) {
-        const participants = _.filter(reportParticipants, email => this.props.currentUserPersonalDetails.login !== email);
-        const hasExcludedIOUEmails = lodashIntersection(reportParticipants, CONST.EXPENSIFY_EMAILS).length > 0;
+    getIOUOptions(participants) {
+        const hasExcludedIOUEmails = lodashIntersection(participants, CONST.EXPENSIFY_EMAILS).length > 0;
```

## Result
<img width="491" alt="Screen Shot 2023-02-03 at 13 41 57" src="https://user-images.githubusercontent.com/113963320/216530311-d3b480a9-291a-438a-ba3e-cecdcd201390.png">

**Comment by bernhardoj** [comment]
### **Updated Proposal**

Updating my previous proposal which lacks some checking. I extract the `getIOUOptions` into a new function placed at `ReportUtils`. Now, it returns array of available IOU options for the report. We can then check if the options contain send or request money to show the text. We can also further use it to check if it contains split bill option to show an alternate text. 

So, both places use the same logic. If there are any logic changes, it will be reflected to both components.

```diff
diff --git a/src/components/ReportWelcomeText.js b/src/components/ReportWelcomeText.js
index c732cc589..4357d6101 100644
--- a/src/components/ReportWelcomeText.js
+++ b/src/components/ReportWelcomeText.js
@@ -15,6 +15,7 @@ import Navigation from '../libs/Navigation/Navigation';
 import ROUTES from '../ROUTES';
 import Tooltip from './Tooltip';
 import reportPropTypes from '../pages/reportPropTypes';
+import CONST from '../CONST';
 
 const personalDetailsPropTypes = PropTypes.shape({
     /** The login of the person (either email or phone number) */
@@ -62,6 +63,8 @@ const ReportWelcomeText = (props) => {
         isMultipleParticipant,
     );
     const roomWelcomeMessage = ReportUtils.getRoomWelcomeMessage(props.report, props.policies);
+    const iouOptions = ReportUtils.getIOUOptions(props.report, participants, props.betas);
+
     return (
         <>
             <View>
@@ -123,10 +126,12 @@ const ReportWelcomeText = (props) => {
                                 {(index < displayNamesWithTooltips.length - 2) && <Text>, </Text>}
                             </Text>
                         ))}
-                        <Text>
-                            {/* Need to confirm copy for the below with marketing, and then add to translations. */}
-                            {props.translate('reportActionsView.usePlusButton')}
-                        </Text>
+                        {(iouOptions.includes(CONST.IOU.IOU_TYPE.SEND) || iouOptions.includes(CONST.IOU.IOU_TYPE.REQUEST)) && (
+                            <Text>
+                                {/* Need to confirm copy for the below with marketing, and then add to translations. */}
+                                {props.translate('reportActionsView.usePlusButton')}
+                            </Text>
+                        )}
+                        // new text here
+                        {(iouOptions.includes(CONST.IOU.IOU_TYPE.SPLIT) && (
+                            <Text>
+                                {props.translate('reportActionsView.usePlusButtonSplit')}
+                            </Text>
+                        )}
                     </Text>
                 )}
             </Text>
@@ -141,6 +146,9 @@ ReportWelcomeText.displayName = 'ReportWelcomeText';
 export default compose(
     withLocalize,
     withOnyx({
+        betas: {
+            key: ONYXKEYS.BETAS,
+        },
         personalDetails: {
             key: ONYXKEYS.PERSONAL_DETAILS,
         },
diff --git a/src/libs/ReportUtils.js b/src/libs/ReportUtils.js
index 51a28759d..902d0c882 100644
--- a/src/libs/ReportUtils.js
+++ b/src/libs/ReportUtils.js
@@ -1,6 +1,7 @@
 import _ from 'underscore';
 import Str from 'expensify-common/lib/str';
 import lodashGet from 'lodash/get';
+import lodashIntersection from 'lodash/intersection';
 import Onyx from 'react-native-onyx';
 import ExpensiMark from 'expensify-common/lib/ExpensiMark';
 import ONYXKEYS from '../ONYXKEYS';
@@ -1371,6 +1372,26 @@ function getNewMarkerReportActionID(report, sortedAndFilteredReportActions) {
         : '';
 }
 
+function getIOUOptions(report, reportParticipants, betas) {
+    const participants = _.filter(reportParticipants, email => currentUserPersonalDetails.login !== email);
+    const hasExcludedIOUEmails = lodashIntersection(reportParticipants, CONST.EXPENSIFY_EMAILS).length > 0;
+    const hasMultipleParticipants = participants.length > 1;
+
+    if (hasExcludedIOUEmails || participants.length === 0 || !Permissions.canUseIOU(betas)) {
+        return [];
+    }
+
+    // User created policy rooms and default rooms like #admins or #announce will always have the Split Bill option
+    // unless there are no participants at all (e.g. #admins room for a policy with only 1 admin)
+    // DM chats and workspace chats will have the Split Bill option only when there are at least 3 people in the chat.
+    if (isChatRoom(report) || hasMultipleParticipants) {
+        return [CONST.IOU.IOU_TYPE.SPLIT];
+    }
+
+    // DM chats and workspace chats that only have 2 people will see the Send / Request money options.
+    return [CONST.IOU.IOU_TYPE.REQUEST, ...(Permissions.canUseIOUSend(betas) ? [CONST.IOU.IOU_TYPE.SEND] : [])];
+}
+
 export {
     getReportParticipantsTitle,
     isReportMessageAttachment,
@@ -1425,4 +1446,5 @@ export {
     getOldDotDefaultAvatar,
     getNewMarkerReportActionID,
     canSeeDefaultRoom,
+    getIOUOptions,
 };
diff --git a/src/pages/home/report/ReportActionCompose.js b/src/pages/home/report/ReportActionCompose.js
index 7ef7fd1d7..99df43f80 100644
--- a/src/pages/home/report/ReportActionCompose.js
+++ b/src/pages/home/report/ReportActionCompose.js
@@ -273,43 +273,25 @@ class ReportActionCompose extends React.Component {
      * @returns {Array<object>}
      */
     getIOUOptions(reportParticipants) {
-        const participants = _.filter(reportParticipants, email => this.props.currentUserPersonalDetails.login !== email);
-        const hasExcludedIOUEmails = lodashIntersection(reportParticipants, CONST.EXPENSIFY_EMAILS).length > 0;
-        const hasMultipleParticipants = participants.length > 1;
-
-        if (hasExcludedIOUEmails || participants.length === 0 || !Permissions.canUseIOU(this.props.betas)) {
-            return [];
-        }
-
-        // User created policy rooms and default rooms like #admins or #announce will always have the Split Bill option
-        // unless there are no participants at all (e.g. #admins room for a policy with only 1 admin)
-        // DM chats and workspace chats will have the Split Bill option only when there are at least 3 people in the chat.
-        if (ReportUtils.isChatRoom(this.props.report) || hasMultipleParticipants) {
-            return [
-                {
-                    icon: Expensicons.Receipt,
-                    text: this.props.translate('iou.splitBill'),
-                    onSelected: () => Navigation.navigate(ROUTES.getIouSplitRoute(this.props.reportID)),
-                },
-            ];
-        }
-
-        // DM chats and workspace chats that only have 2 people will see the Send / Request money options.
-        return [
-            {
+        const options = {
+            [CONST.IOU.IOU_TYPE.SPLIT]: {
+                icon: Expensicons.Receipt,
+                text: this.props.translate('iou.splitBill'),
+                onSelected: () => Navigation.navigate(ROUTES.getIouSplitRoute(this.props.reportID)),
+            },
+            [CONST.IOU.IOU_TYPE.REQUEST]: {
                 icon: Expensicons.MoneyCircle,
                 text: this.props.translate('iou.requestMoney'),
                 onSelected: () => Navigation.navigate(ROUTES.getIouRequestRoute(this.props.reportID)),
             },
-            ...(Permissions.canUseIOUSend(this.props.betas)
-                ? [
-                    {
-                        icon: Expensicons.Send,
-                        text: this.props.translate('iou.sendMoney'),
-                        onSelected: () => Navigation.navigate(ROUTES.getIOUSendRoute(this.props.reportID)),
-                    }]
-                : []),
-        ];
+            [CONST.IOU.IOU_TYPE.SEND]: {
+                icon: Expensicons.Send,
+                text: this.props.translate('iou.sendMoney'),
+                onSelected: () => Navigation.navigate(ROUTES.getIOUSendRoute(this.props.reportID)),
+            },
+        };
+        return ReportUtils.getIOUOptions(this.props.report, reportParticipants, this.props.betas)
+            .map(option => options[option]);
     }
 
     /**

```

**Result**

https://user-images.githubusercontent.com/50919443/216551526-86c59a32-63f6-445f-91da-631e4997d6f9.mp4

**Comment by MelvinBot** [comment]
@thienlnam, @Christinadobrzyn, @aimane-chnaif Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by thienlnam** [comment]
@aimane-chnaif Could you please review these proposals?

**Comment by hungvu193** [comment]
<b>Proposal</b>
Here's the cases that I think we shouldn't display the "You can also use the + button below to send or request money!" text (or at least we need to update that with another message).
- Multiple participants report.
- Report that includes Concierge.
- Report that has error.

<b>Solution</b>
Add condition to ReportWelcomeText, whether to show the `usePlusButton` message or not.
```diff
diff --git a/src/components/ReportWelcomeText.js b/src/components/ReportWelcomeText.js
index c732cc589..f282c4912 100644
--- a/src/components/ReportWelcomeText.js
+++ b/src/components/ReportWelcomeText.js
@@ -52,6 +52,11 @@ const defaultProps = {
 };
 
 const ReportWelcomeText = (props) => {
+    const addWorkspaceRoomOrChatErrors = lodashGet(props.report, 'errorFields.addWorkspaceRoom')
+        || lodashGet(props.report, 'errorFields.createChat');
+
+    const hasError = !_.isEmpty(addWorkspaceRoomOrChatErrors);
+
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const isChatRoom = ReportUtils.isChatRoom(props.report);
     const isDefault = !(isChatRoom || isPolicyExpenseChat);
@@ -61,6 +66,10 @@ const ReportWelcomeText = (props) => {
         OptionsListUtils.getPersonalDetailsForLogins(participants, props.personalDetails),
         isMultipleParticipant,
     );
+    const hideRequestMoney = isMultipleParticipant
+      || hasError
+      || ReportUtils.chatIncludesConcierge(props.report);
+
     const roomWelcomeMessage = ReportUtils.getRoomWelcomeMessage(props.report, props.policies);
     return (
         <>
@@ -123,10 +132,12 @@ const ReportWelcomeText = (props) => {
                                 {(index < displayNamesWithTooltips.length - 2) && <Text>, </Text>}
                             </Text>
                         ))}
+                        {!hideRequestMoney && (
                         <Text>
                             {/* Need to confirm copy for the below with marketing, and then add to translations. */}
                             {props.translate('reportActionsView.usePlusButton')}
                         </Text>
+                        )}
                     </Text>
                 )}
             </Text>

```
Result:

https://user-images.githubusercontent.com/16502320/217149679-ac344143-117b-424a-80c1-0ebfc5840e3d.mov

**Comment by thienlnam** [comment]
Thanks for the reviews @aimane-chnaif - agree with your analysis

**Comment by MelvinBot** [comment]
📣 @bernhardoj You have been assigned to this job by @thienlnam!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by bernhardoj** [comment]
I will open the PR in a couple of hours.

**Comment by bernhardoj** [comment]
PR is ready for review.

**Comment by thienlnam** [comment]
@aimane-chnaif Let's just keep it as is for now, once we move out of the IOU send beta we'll have to change it back anyways

**Comment by Christinadobrzyn** [comment]
Hired: 

- @dhanashree-sawant as report $250
- @aimane-chnaif as C+
- @bernhardoj for the fix

Internal Upwork profile: https://www.upwork.com/ab/applicants/1621383713253961728/job-details

**Comment by dhanashree-sawant** [comment]
@Christinadobrzyn thanks, offer accepted.

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.74-0 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/15162


If no regressions arise, payment will be issued on 2023-02-28. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@aimane-chnaif / @thienlnam] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@aimane-chnaif / @thienlnam] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@aimane-chnaif / @thienlnam] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@bernhardoj] Propose [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@Christinadobrzyn] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by Christinadobrzyn** [comment]
Sorry for the delay in payment here! 

- Looks like the PR was [created Feb 15th](https://github.com/Expensify/App/issues/14764#issuecomment-1431005768) and [merged Feb 16th](https://github.com/Expensify/App/pull/15162#issuecomment-1432411806) so that's within our bonus timeframe. 
- Paid @dhairyasenjaliya as reporter
- Paid @aimane-chnaif + 50% bonus as C+
- Paid @bernhardoj + 50% bonus as contributor

All that's left is the [checklist](https://github.com/Expensify/App/issues/14764#issuecomment-1439036546) @bernhardoj let me know when you have the regression test proposal and I'll get that over to our QA. Thanks!

**Comment by bernhardoj** [comment]
Sorry, I totally forget about the regression test 😅.

Regression Test Proposal

1. Open a chat of:
a. Concierge
b. Group
b. Policy room
c. Default room
d. Policy expense chat (> 2 members)
2. Verify the `You can also use the + button below to send or request money!` text at the welcome text does not show
3. Open a chat of:
a. DM
b. Policy expense chat (2 members)
3. Verify the `You can also use the + button below to send or request money!` text at the welcome text shows

Do we agree 👍 or 👎

**Comment by Christinadobrzyn** [comment]
I need to do some testing on this regression proposal as I'm not sure where to add it to test rail - I'll work on that tomorrow

**Comment by Christinadobrzyn** [comment]
Here's the proposals for the regression tests. 

Regression test related to the chat; 

Regression Test Buddy Check
Context: When you create a new chat you should see `You can also use the + button below to send or request money!` on the welcome page 
GitHub link: [here](https://github.com/Expensify/App/issues/14764)
Notes: Let’s update the `Create new 1:1 conversation - Creation and LHN` test
Recommendation: Update the existing flow to add these steps
Please :yes: or :no: if you think this is the right path forward! Thank you!

<img width="1048" alt="image" src="https://user-images.githubusercontent.com/51066321/223659839-7e417d7b-13bf-4bfe-9791-ddeb7f563c20.png">

1. Search for a user that you don't have any messages with and open the chat. (existing)
2. Send a message with the text: STAGING_CHAT_MESSAGE_A2C534B7-3509-416E-A0AD-8463831C29DD. (existing)
3. **Verify** that when you switch away from the chat, the chat doesn't disappears from left hand navigation. (existing)
4. **Verify** that in the left hand navigation chat list, a preview of the last message in the conversation is displayed under account B. (existing)
5. Verify you see `You can also use the + button below to send or request money! `on the welcome screen of the chat (new)


Related to Group chats

Regression Test Buddy Check
Context: When you create a new chat you should see `You can also use the + button below to send or request money!` on the welcome page 
GitHub link: [here](https://github.com/Expensify/App/issues/14764)
Notes: Let’s update the `Create Group Conversation` test
Recommendation: Update the existing flow to add these steps
Please :yes: or :no: if you think this is the right path forward! Thank you!

<img width="1101" alt="image" src="https://user-images.githubusercontent.com/51066321/223661074-09cf795f-9d6d-473a-bee9-97569fc6b120.png">

1. Search for 4 users and add them to the group. Be sure that you have access to at least 2 of them (existing)
3. Verify when selecting users, they are displayed on the top of the list with a green check mark on the right (existing)
4. Click on create group (existing)
5. Verify that the group is created with all the participants you selected in the list. You can confirm this by clicking on the header to view the group information. (existing)
6. **Verify** you don't see `You can also use the + button below to send or request money! `on the welcome screen of the Group chat (new)

Related to Rooms

Regression Test Buddy Check
Context: When you create a new chat you should see `You can also use the + button below to send or request money!` on the welcome page 
GitHub link: [here](https://github.com/Expensify/App/issues/14764)
Notes: Let’s update the `Room - Create Room Conversation (Online/Offline)` [test](https://expensify.testrail.io/index.php?/cases/view/1971088&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=229067)
Recommendation: Update the existing flow to add these steps
Please :yes: or :no: if you think this is the right path forward! Thank you!

<img width="1076" alt="image" src="https://user-images.githubusercontent.com/51066321/223661893-bd22130a-2f03-4085-a74b-88958fef6750.png">

1. Send a message in the #Room and Verify the message is correctly displayed in the conversation history. (existing)
2. Tap on the header of the #Room and Verify it displays: Members, Settings, Invite and Leave Room. (existing)
3. **Verify** you don't see `You can also use the + button below to send or request money! `on the welcome screen of the Group chat (new)
4. Log in as another account that has access to the same workspace (if needed invite another account you have access) (existing)

**Comment by MelvinBot** [comment]
@thienlnam, @bernhardoj, @Christinadobrzyn, @aimane-chnaif Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by Christinadobrzyn** [comment]
Sorry this is on me, still waiting for a buddy check on the regression tests. Bumped my Slack posts.
